ATAPHtmlReport/ATAPHtmlReport.psm1

```diff
@@ -818,20 +824,23 @@ function ConvertTo-HtmlTable {
         htmlElement 'tbody' @{id='MITREtbody'} {
             htmlElement 'tr' @{} {
                 foreach ($tactic in $Mappings.Keys) {
-                    htmlElement 'td' @{id='MITREtbody'} {
+                    htmlElement 'td' @{id='MITREtbody'; class =$colorClass} {
```

Comment:
die variable colorClass ist an der stelle noch gar nicht initialisiert, dementsprechend ist die am ende auch gar nicht in der html, sollte also entfernt werden